bucket/capnp.json

```
@@ -0,0 +1,25 @@
+{
+    "version": "1.0.2",
+    "description": "Capnp Proto is an insanely fast data interchange format and capability-based RPC system",
```

```suggestion
    "description": "Cap'n Proto is an insanely fast data interchange format and capability-based RPC system",
```
Suggestion to fix the name in the description